src/main/java/ch/wisv/events/core/model/sales/Vendor.java

```diff
@@ -34,54 +32,43 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+@Data
 @Entity
-@EqualsAndHashCode
 public class Vendor {
 
     /**
      * Field id of the vendor.
      */
     @Id
     @GeneratedValue
-    @Getter
     private Integer id;
```

Comment:
You can use `@Setter(AccesLevel.NONE)` to prevent a setter being generated by Lombok